deploy/osd-curated-operatorsources-revert/fedramp/00-operatorhub.cr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: config.openshift.io/v1
```

Comment:
Please place these files in a separate directory under deploy. Something like `deploy/fedramp-operatorhub/` or the like. Don't want this getting mixed into the curated operatorsources stuff (it's really legacy)

Reply:
moved to its own folder